packages/backend-ai/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { Message, Prompt, Tool, ContentBlock } from './types.js';
+
+export { getMessage } from './get_message.js';
```

Comment:
Is it conventional in this repo to import from the js files?

Reply:
yes